shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/docker/proxy/conf/readwrite_splitting/h2/config-readwrite-splitting.yaml

```diff
@@ -19,7 +19,7 @@ schemaName: readwrite_splitting
 
 dataSources:
   write_ds:
-    url: jdbc:mysql://mysql.readwrite_splitting.host:3306/write_ds?serverTimezone=UTC&useSSL=false&characterEncoding=utf-8
+    url: jdbc:h2://h2.readwrite_splitting.host:3306/write_ds?serverTimezone=UTC&useSSL=false&characterEncoding=utf-8
```

Comment:
Please use real h2 jdbc url, it looks like `jdbc:h2:mem:db_name;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false;MODE=MYSQL`